expo/gbe/report_views.py

```diff
@@ -273,11 +296,12 @@ def export_act_techinfo(request, show_id):
             techinfo.append(start)
 
     # end for loop through acts
-    cuesequenceindex = 24 #  magic number, obtained by counting headers
+    cuesequenceindex = 24  # magic number, obtained by counting headers
 
     techinfo = sorted(techinfo, key=lambda row: row[0])
     response = HttpResponse(content_type='text/csv')
-    response['Content-Disposition'] = 'attachment; filename=%s_acttect.csv' % show.title.replace(' ','_')
+    response['Content-Disposition'] = 'attachment; filename=%s_acttect.csv' \
```

Comment:
Not a blocker, but I prefer to use parens rather than backslash continuations:
`response['Content-Disposition'] = ('attachment; filename=%s_acttect.csv' % 
                               show.title.replace(' ', '_'))`